tests/ci/azureml_tests/test_groups.py

```diff
@@ -426,13 +426,13 @@
     ],
     "group_gpu_002": [  # Total group time:
         "tests/unit/examples/test_notebooks_gpu.py::test_gpu_vm",  # 0.76s (Always the first test to check the GPU works)
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_xdeepfm_component_definition",  # FIXME: Issue with TF version > 2.10.1 See #2018
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_dkn_component_definition",
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_dkn_item2item_component_definition",
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_slirec_component_definition",  # FIXME: Issue with TF version > 2.10.1 See #2018
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_nextitnet_component_definition",
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_sum_component_definition",  # FIXME: Issue with TF version > 2.10.1 See #2018
-        "tests/unit/recommenders/models/test_deeprec_model.py::test_lightgcn_component_definition",
+        # "tests/unit/recommenders/models/test_deeprec_model.py::test_xdeepfm_component_definition",  # FIXME: Disabled due to the issue with TF version > 2.10.1 See #2018
+        # "tests/unit/recommenders/models/test_deeprec_model.py::test_dkn_component_definition",  # FIXME: Disabled due to the issue with TF version > 2.10.1 See #2018
+        # "tests/unit/recommenders/models/test_deeprec_model.py::test_dkn_item2item_component_definition",  # FIXME: Disabled due to the issue with TF version > 2.10.1 See #2018
+        # "tests/unit/recommenders/models/test_deeprec_model.py::test_slirec_component_definition",  # FIXME: Disabled due to the issue with TF version > 2.10.1 See #2018
+        # "tests/unit/recommenders/models/test_deeprec_model.py::test_nextitnet_component_definition",  # FIXME: Disabled due to the issue with TF version > 2.10.1 See #2018
```

Comment:
@SimonYansenZhao I thought that the issue was only with xDeepFM. All the algos in deeprec are affected?

Reply:
Not sure.  I'll check.

Reply:
Actually, I don't see any class that uses `SUMModel` (which is the class that causes the error). There are a couple of tests that reference it (including the one above), however. So maybe it is not a dependency and has been left somehow in the code? If so, then we could simply remove the tests without breaking any of the notebooks.

Reply:
Uncommeted